### PR TITLE
Minor UI and error reporting fixes

### PR DESCRIPTION
**Pull request by bderenzi**

### CODE REVIEW

**Review by kaapstorm — APPROVED** [review]

File: mpact/services.py [hunk 1]
```diff
@@ -284,8 +293,8 @@ def is_blank(val):
             days = row["Days"]
             message = row["Message"]
             comment = row["Comment"] or ''
-            if is_blank(days) or is_blank(message):
-                bad_rows[sheet["title"]].append(n)
+            if is_blank(days) or is_blank(message) or not is_positive_numeric(days):
+                bad_rows[sheet["title"]].append(n+1)
```
Comment (kaapstorm) [inline mpact/services.py:297]: Are you sure this is `n + 1`? `enumerate()` on line 292 is starting at 1 already. Should it start at 2?

Comment (bderenzi) [reply mpact/services.py:297]: I probably should have added a comment there. 

The `enumerate()` starts at 1, which means it skips the 'first' row, which is the header. The `n+1` here is for the UI response to tell the user which row is incorrect. Excel / humans don't dp 0-indexed arrays, so added one to make it more clear which row has the problem.

File: mpact/services.py [hunk 2]
```diff
@@ -262,6 +262,15 @@ def schedule_messages(xlsx_file):
 
     def is_blank(val):
         return val is None or val == ''
+    
+    def is_positive_numeric(val):
+        try:
+            val=int(val)
+            if val >= 0:
+                return True
```
Comment (kaapstorm) [inline mpact/services.py:270]: I'm being petty. I know. But I can't stop looking at it.
```suggestion
            return int(val) >= 0
```

File: mpact/services.py [hunk 3]
```diff
@@ -262,6 +262,15 @@ def schedule_messages(xlsx_file):
...
+        except ValueError:
+            pass
+        return False
```
Comment (czue) [inline mpact/services.py:273]: nit: could also put this in the except clause

Comment (bderenzi) [reply mpact/services.py:273]: Good point. I think that wasn't true before @kaapstorm 's suggestion of checking that it is a positive integer in a single return statement.

**Pull request merged**